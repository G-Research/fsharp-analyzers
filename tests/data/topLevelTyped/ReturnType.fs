module RT

let a (b : int) = b + 1

let c = 'c'

type T =
    member _.Foo (bar : int) = bar + 4 
    member _.Char = 'd'
