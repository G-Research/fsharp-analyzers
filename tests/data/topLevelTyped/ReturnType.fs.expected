GRA-UNTYPEDAPI-001 | Warning | (3,4 - 3,5) | `a` is not private, not fully typed and could be made private to file.
Specify the return type.
GRA-UNTYPEDAPI-001 | Warning | (5,4 - 5,5) | `c` is not private, not fully typed and could be made private to file.
Specify the return type.
GRA-UNTYPEDAPI-001 | Warning | (8,13 - 8,16) | `Foo` is not private, not fully typed
Specify the return type.
GRA-UNTYPEDAPI-001 | Warning | (9,13 - 9,17) | `Char` is not private, not fully typed
Specify the return type.
